compiler+runtime/src/jank/clojure/core.jank

```diff
@@ -4404,7 +4404,9 @@
   values as thread-local bindings. Then calls f with the supplied arguments.
   Pops the installed bindings after f returned. Returns whatever f returns."
   [binding-map f & args]
-  (push-thread-bindings binding-map)
+  (push-thread-bindings (if (= "persistent_array_map" (type binding-map))
+                          (into {} binding-map)
```

Comment:
Is this meant to put it into a hash map? `(type {})` is also `"persistent_array_map"`. Could use `(hash-map)`.

Reply:
Yes, I'm trying to convert all input maps to a `"persistent_hash_map"`. In my local testing `(type (into {} {:x :y})) => "persistent_hash_map"` but maybe that's an implementation detail we shouldn't leverage here. Are you thinking something like `(into (hash-map) {:x :y})` is better here?

Reply:
Yep, I think that's safer and more explicit. I'm surprised `(into {} ...)` gives a hash map.

Reply:
Done 👍